azure/functions/http.py

```diff
@@ -46,16 +48,18 @@ def get_body(self) -> bytes:
         return self.__body_bytes
 
     def get_json(self) -> typing.Any:
-        if (self.__body_type == 'json'
-                or self.__body_type == 'string'):
+        if self.__body_type in ('json', 'string'):
             assert self.__body_str is not None
             return json.loads(self.__body_str)
-        else:
+        elif self.__body_bytes is not None:
             try:
                 return json.loads(self.__body_bytes.decode())
             except ValueError as e:
                 raise ValueError(
                     'HTTP request does not contain valid JSON data') from e
+        else:
+            raise ValueError(
+                'Request body cannot be empty in JSON deserialization') from e
```

Comment:
Would it help if we could also return what we got in __body_bytes? Also, for `json.loads(self.__body_bytes.decode())`, do we need to make `utf-08` explicit during the decode phase?